commandreporter/command_reporter_test.go

```diff
@@ -66,10 +65,9 @@ var _ = Describe("CommandReporter", func() {
 			reporter = commandreporter.NewCommandReporter(writer)
 			t = time.Date(2009, time.November, 10, 23, 0, 0, 0, time.UTC)
 			timestampRegex = "\\[2009-11-10 23:00:00.00 \\(UTC\\)\\]>"
-			config.DefaultReporterConfig.NoColor = false
```

Comment:
I think this could be changed to something like the following:
```golang
_, reporterConfig := GinkgoConfiguration()
reporterConfig.NoColor = false
```

See [overriding Ginkgo's command-line configuration in the suite](https://onsi.github.io/ginkgo/#overriding-ginkgos-command-line-configuration-in-the-suite) in the docs for reference.

Reply:
Actually... this bit is more complicated. We'd have to pass the new `ReporterConfig` to the `RunSpecs()` function.

Reply:
it is complicated.  Though I solved it in a way I don't like but is simple.  Curious your thoughts when you next review